mitielib/mitie.py

```diff
@@ -649,26 +672,38 @@ def train(self):
 
 
 class text_categorizer:
-    def __init__(self, filename):
+    def __init__(self, filename, fe_filename=None):
         filename = to_bytes(filename)
         self.__mitie_free = _f.mitie_free
-        if isinstance(filename, ctypes.c_void_p):
+        if (isinstance(filename, ctypes.c_void_p)):
             self.__obj = filename
         else:
-            self.__obj = _f.mitie_load_text_categorizer(filename)
-        if self.__obj is None:
-            raise Exception("Unable to load text_categorizer detector from " + filename)
+            if (fe_filename is None):
+                self.__obj = _f.mitie_load_text_categorizer(filename)
+            else:
+                print(filename,fe_filename)               
+                self.__obj = _f.mitie_load_text_categorizer_pure_model(filename,fe_filename)
+        if (self.__obj == None):
+            raise Exception("Unable to load text_categorizer detector from " + to_default_str_type(filename))
 
     def __del__(self):
         self.__mitie_free(self.__obj)
 
-    def save_to_disk(self, filename):
+    def save_to_disk(self, filename,pure_model=False):
         """Save this object to disk.  You recall it from disk with the following Python
         code: 
-            tcat = text_categorizer(filename)"""
+            tcat = text_categorizer(filename)
```

Comment:
This is also a little confusing.  The way it's worded makes it sound as if 'you must pass a feature extractor' when saving in pure mode.